### PR TITLE
[CDAP-15795] New CDAP home page

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-15795
Build: https://builds.cask.co/browse/CDAP-UDUT401
E2E: https://builds.cask.co/browse/IT-UPD275

### CODE REVIEW

File: cdap-ui/cypress/integration/nuxtour.spec.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import * as Helpers from '../helpers';
 
 let headers = {};
-const controlCenterText = 'Control Center';
+const studioText = 'Pipeline Studio';
 
 // Assumes 5 step tour: Control Center, Pipeline Studio, Wrangler, Metadata, Hub
```
Comment (yukiej) [inline cdap-ui/cypress/integration/nuxtour.spec.ts]: Update the order of the steps in comment so that it starts with pipeline studio?

File: cdap-ui/app/cdap/components/Home/HomeActions/ActionConfig.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    links: [
+      {
+        label: 'Dashboard',
+        url: '/cdap/ns/:namespace/operations',
+      },
+    ],
+  },
+  {
+    img: '/cdap_assets/img/administration.svg',
+    title: 'Administrations',
```
Comment (yukiej) [inline cdap-ui/app/cdap/components/Home/HomeActions/ActionConfig.ts]: Should probably be "Administration" without the s to match rest of UI?

File: cdap-ui/app/cdap/components/Home/HomeActions/ActionConfig.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+export const ActionConfig = [
+  {
+    img: '/cdap_assets/img/cleanse_data.svg',
+    title: 'Cleanse data',
```
Comment (yukiej) [inline cdap-ui/app/cdap/components/Home/HomeActions/ActionConfig.ts]: i18n for card text?

File: cdap-ui/app/cdap/components/Home/HomeActions/ActionConfig.ts (hunk 2 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/Home/HomeActions/ActionConfig.ts]: I'm not sure about this. @leacd any input?

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 4]
```diff
@@ -1678,6 +1678,30 @@ features:
       timeRangeTitle: Select time range
       to: To
 
+  Home:
+    Administrations:
+      description: Manage system and namespace level settings and preferences.
+      namespaceLabel: Namespace
+      systemLabel: System
+      title: Administrations
```
Comment (yukiej) [inline cdap-ui/app/cdap/text/text-en.yaml]: Pretty sure it is more common to write without the 's' at the end -  I haven't seen it with an 's' before.  We do this in our existing UI as well. 

Screen Shot 2019-09-09 at 12 09 31 PM

**Review by yukiej — APPROVED** [review]
One small comment about wording on one of the cards. Otherwise LGTM!

File: cdap-ui/app/cdap/components/Home/HomeActions/ActionCard.tsx [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+        <div className={classes.description}>{config.description}</div>
+      </div>
+      <div className={classes.footer}>
+        {config.links.slice(0, 2).map((link) => {
+          const url = link.url.includes(':')
+            ? buildUrl(link.url, { namespace: getCurrentNamespace() })
+            : link.url;
+
+          return (
+            <a href={url} key={link.label} className={classes.link}>
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Home/HomeActions/ActionCard.tsx]: This should be a `Link` for links in react right? Right now this will reload the entire page even when navigating from Home to say Wrangler?

File: cdap-ui/app/cdap/components/Home/Welcome/index.tsx [hunk 6]
```diff
@@ -40,7 +40,7 @@ interface IWelcomeState {
   showAgain: boolean;
 }
 
-export default class Welcome extends React.PureComponent<void, IWelcomeState> {
+export default class Welcome extends React.PureComponent<{}, IWelcomeState> {
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Home/Welcome/index.tsx:43]: Just curious why is this change needed?

**Review by ajainarayanan — APPROVED** [review]
Couple of minor comments. Otherwise 👍 LGTM

File: cdap-ui/app/cdap/components/Home/Welcome/index.tsx (hunk 6 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/Home/Welcome/index.tsx:43]: void is not allowed for Props type apparently. It was throwing tslint error.

File: cdap-ui/app/cdap/components/Home/HomeActions/ActionCard.tsx (hunk 5 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/Home/HomeActions/ActionCard.tsx]: I'm not sure how I can differentiate react vs angular from a generic component. Any suggestion?

Comment (elfenheart) [reply cdap-ui/app/cdap/components/Home/HomeActions/ActionCard.tsx]: added a check if link url starts with `/cdap`
